corehq/apps/es/migrations/0001_add_tombstones.py

```diff
@@ -0,0 +1,69 @@
+# Generated by Django 3.2.16 on 2023-01-16 15:31
+
+import corehq.apps.es.migration_operations
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        corehq.apps.es.migration_operations.UpdateIndexMapping(
+            name='hqapps_2020-02-26',
+            type_='app',
+            properties={
+                '__is_tombstone__': {'type': 'boolean'},
+            },
+        ),
+        corehq.apps.es.migration_operations.UpdateIndexMapping(
+            name='case_search_2018-05-29',
```

Comment:
This will need to conditionally come from Django settings. See [here](https://github.com/dimagi/commcare-hq/blob/c201e2f5c20/corehq/apps/es/case_search.py#L158).